Contents/mods/Hydrocraft/media/scripts/Chemistry.txt

```diff
@@ -185,6 +206,7 @@ item HCPhosphoruspowder
         Type            = Normal,
         DisplayName             = Phosphorus Powder,
         icon            = HCPhosphoruspowder,
+		DisplayCategory	= CraftMine,
```

Comment:
maybe drugs? since its a precursor for meth only

Reply:
I tried to only mark things that can be used as Drugs not the mats (And the direkt box variants for sorting convenience). So i will put it to CraftChem.